sdk/core/core/src/az_http_policy.c

```diff
@@ -49,11 +49,11 @@ AZ_NODISCARD az_result az_http_pipeline_policy_uniquerequestid(
   (void)p_options;
 
   // TODO - add a UUID create implementation
-  az_span const uniqueid = AZ_SPAN_LITERAL_FROM_STR("123e4567-e89b-12d3-a456-426655440000");
+  //az_span const uniqueid = AZ_SPAN_LITERAL_FROM_STR("123e4567-e89b-12d3-a456-426655440000");
```

Comment:
nit: Can we remove the code and file an issue linking back to this PR for the TODO rather than commenting it out? We always have code commit history, but haven't commented out code makes it seem like someone accidentally left "dead" code around. Alternatively, explicitly state why this code is commented out, so it doesn't look like "real" C code.